tests/integration/transformers/test_boolean.py

```diff
@@ -85,3 +86,24 @@ def test_boolean_missing_value_generation_none(self):
         expected_reversed = pd.DataFrame({'bool': [True, True, True, False]})
         pd.testing.assert_frame_equal(transformed, expected_transformed)
         pd.testing.assert_frame_equal(reverse, expected_reversed, check_dtype=False)
+
+    def test__reverse_transform_from_manually_set_parameters(self):
+        """Test the ``reverse_transform`` after manually setting parameters."""
+        # Setup
+        data = pd.DataFrame([True, True, None, False], columns=['bool'])
+        transformed = pd.DataFrame({
+            'bool': [1.0, 1.0, 1.0, 0.0],
+            'bool.is_null': [0.0, 0.0, 1.0, 0.0],
+        })
+        column_name = 'bool'
+        transformer = BinaryEncoder()
+
+        # Run
+        null_transformer = NullTransformer('mode')
+        transformer.reset_randomization()
+        transformer._set_fitted_parameters(column_name, null_transformer)
+        reverse = transformer.reverse_transform(transformed)
+        reverse[column_name] = np.where(reverse['bool.is_null'] == 1.0, None, reverse[column_name])
```

Comment:
We shouldn't have to do this step here since the BinaryEncoder has a built-in NullTransformer, but we do since we only populate `self.output_columns` with the column name. Instead of modifying it here, can you try appending `'bool.is_null'` to the `output_columns` parameter?

Reply:
Moved it but I had to call `_set_fitted_parameters` for the NullTransformer. I also added another test for `from_column` and `random` based NullTransformers